net/mwan3/Makefile

```diff
@@ -8,9 +8,9 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=mwan3
-PKG_VERSION:=2.0
-PKG_RELEASE:=3
-PKG_MAINTAINER:=Jeroen Louwes <jeroen.louwes@gmail.com>
+PKG_VERSION:=2.0.1
+PKG_RELEASE:=4
```

Comment:
agreed, if code is different, version MUST be different so it's easier for bug reports